models/users-models.js

```diff
@@ -0,0 +1,10 @@
+const db = require("../db/connection");
+
+exports.selectUsers = () => {
+  return db.query("SELECT * FROM users").then((result) => {
+    if (result.rows.length < 1) {
```

Comment:
remove this conditional logic because if there are no users available we want to send back a 200 with an empty array